### DISCUSSION TITLE
Contributor call: 14 May 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2155] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lexander2002. Sorry, no spiking the punch this time 😔😔
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 30 April 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/apr-30-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Jacqueline's feedback
- Migration from Netlify to Vercel
- How to avoid having to recreate the universe prior to deploying a website
- Changes in dev/lab sites as a precursor to the new org site
- Proposed backend architectural changes (mongo -> sqlite)
- Wanted: UI/UX designers

> This discussion will be updated after the call with notes and to-dos.

**Comment by l3xander** [comment, marked as answer]
### Notes
- @joostdecock: we are migrating from Netlify to Vercel.
    - Why? Vercel gives a few more features. They are sponsoring us too.
    - If you’ve experienced problems recently, it's probably because a bug caused that we temporarily got dropped to the Hobby tier (instead of staying at the sponsored Pro tier).
    - We’re starting with the .lab and .dev sites first.
    - (Note to @joostdecock: @bobgeorgethe3rd, Jackseye on Discord and @starfetch would like access.)
- @joostdecock: to follow up on the previous point, deploying a site with Vercel takes a long time. (For example, it took 25 minutes to load the .lab site.) We may need to make changes to the monorepo so we can do quick builds without having to recreate ✨the FreeSewing universe✨
- @joostdecock: there are some small changes to the .dev and .lab sites as a precursor to the new .org site, to name a few: fixes, loading animation, auto generated table of contents, and a flashy new gradient.
- @joostdecock: we’re switching over from MongoDB to SQLite. The biggest advantage of SQLite is that developers can more easily spin up their own instance of the backend, which gives a better workflow.
- Wanted: UI/UX designers!
    - @joostdecock: we want to make the sites easier to understand, more consistent and more intuitive. Anyone who has opinions on what the site should look like, is more than welcome to give feedback. You don’t need to know how to implement it.
    - @tangerineshark added that we can put out a call in one of our Discord channels.
    - @joostdecock recommends [Maximillian Schwarzmüller](https://www.udemy.com/user/maximilian-schwarzmuller/) to those who do want to learn how to implement this.
- @starfetch is working on a cloak and may add options for different lengths with the feature ‘optional measurements’.
- I (@lexander2002) withdrew from being interviewed for the upcoming newsletter. Now the question is, who’s next? There are a lot of great candidates: we can interview multiple people to ‘stock up’ on interviews.
- Bill from Discord suggested we stream the Contributor Calls to Twitch or YouTube to reach more people.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
